templates/base.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>{{ title }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- Genearl Stylesheets -->
+    <link rel="stylesheet" href="/static/styles/eventslyStyleSheet.css">
+    <link rel="stylesheet" href="/static/styles/navbarStyleSheet.css">
+    <link rel="stylesheet" href="/static/styles/footerStyleSheet.css">
+
+    <!-- Specific Styles NOTE: (variable shows as error, but it is not)-->
+    <style type="text/css">{{ pageStyles }}</style>
+
+    <!-- Eventsly Scripts -->
+    <script src='/static/scripts/navScripts.js' async></script>
+
+    <!-- Font Awesome JS load for Eventsly Logo -->
+    <script src="https://kit.fontawesome.com/3f6d9190cb.js" crossorigin="anonymous"></script>
+
+    <!-- Google Font API load for Eventsly Logo -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400&display=swap" rel="stylesheet">
+
+    <!-- block for page-specific head content (stylesheets, scripts, etc) -->
+    {% block head %} {% endblock %}
+</head>
+
+<body>
+    <header>
+        <!-- .active class is applied dynamically based title variable (specified in app.py render_tempate code) -->
+        <nav role="navigation">
```

Comment:
The navbar code is now applied the exact same way to each .html page, with the exception of the `active` class (which makes the link of the current page white instead of purple). The `active` class is applied to the `nav-item` based on the `title` variable, e.g., when the `title` is "Bookmarks", the "Bookmarks" link will be white in the navbar.